scipy/misc/pilutil.py

```diff
@@ -29,7 +29,7 @@
            'imrotate', 'imresize', 'imshow', 'imfilter']
 
 
-# Returns a byte-scaled image
+@numpy.deprecate(message="`bytescale` is deprecated in SciPy 1.0.0.")
```

Comment:
I think two releases is reasonable. So how about "... and will be removed in 1.2.0", assuming others agree.

Reply:
Two releases is our default policy, so sounds reasonable to me to start stating that explicitly for all deprecations.